sample-lnd.conf

```diff
@@ -963,6 +963,37 @@
 ; Instructs lnd to encrypt the private key using the wallet's seed.
 ; tor.encryptkey=false
 
+[logging]
+
+; Disable logging to stdout and stderror.
+; Default:
+;   logging.console.disable=false
+; Example:
+;   logging.console.disable=true
+
+; Don't add timestamps to logs written to stdout and stderr.
+; Default:
+;   logging.console.no-timestamps=false
+; Example:
+;   logging.console.no-timestamps=true
+
+; Set color theme for to stdout and stderror logger.
```

Comment:
Should we list all possible values here? Since the `choice` is only rendered on the console when printing the help text.

Reply:
converted this to a bool now